src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -237,16 +237,23 @@ export class MdProgressSpinner implements OnDestroy {
 
   /**
    * Removes interval, ending the animation.
+   * @param redraw Whether to redraw the circle after the animation is cleared.
    */
-  private _cleanupIndeterminateAnimation() {
-    this.interdeterminateInterval = null;
+  private _cleanupIndeterminateAnimation(redraw = false) {
```

Comment:
Rather than adding a `redraw` option here and then calling `_animateCircle`, can we instead just call `_animateCircle` after  `_cleanupIndeterminateAnimation` in the setter? I generally like to avoid boolean parameters that tell a function to do extra work, and the call site doesn't make it obvious what the parameter is for.